library/src/main/res/layout/mapbox_mapview_internal.xml

```diff
@@ -49,29 +50,62 @@
     <!-- This is the target button shown on the bottom-right -->
     <ImageButton
         android:id="@+id/ib_mapview_focusOnMyLocationIcon"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:src="@drawable/ic_my_location"
-        android:layout_marginBottom="20dp"
+        android:layout_width="70dp"
+        android:layout_height="70dp"
+        android:src="@drawable/ic_cross_hair"
+        android:layout_marginTop="20dp"
         android:layout_marginRight="20dp"
-        android:layout_gravity="bottom|right"
+        android:layout_gravity="top|right"
```

Comment:
I think we want this on the bottom-right or left and probably provide customisation ability for it's position and icon. Follow more about this [here](https://onaio.slack.com/archives/C5DAYMPH6/p1539882288000100) . I think we can use [this version of OMK](https://play.google.com/store/apps/details?id=org.redcross.openmapkit&hl=en) as a reference

![screenshot_2018-11-09-15-21-25-667_org redcross openmapkit](https://user-images.githubusercontent.com/31766075/48262660-6fcdf100-e434-11e8-9f00-7dfe84901e31.png)

Reply:
As per this conversation https://onaio.slack.com/archives/CBSU8L2SC/p1542029741008000 the current ui implementation is correct.